packages/bundler-utils/src/https.ts

```diff
@@ -16,7 +16,7 @@ export type { Server as SpdyServer } from 'spdy';
 export async function resolveHttpsConfig(httpsConfig: HttpsServerOptions) {
   // Check if mkcert is installed
   try {
-    await execa.execa('mkcert', ['--version']);
+    await execa.execa('mkcert', ['-help']);
```

Comment:
```suggestion
    // use mkcert -help instead of mkcert --version for checking if mkcert is installed, cause mkcert --version does not exists in Linux
    await execa.execa('mkcert', ['-help']);
```